proto/babylon/checkpointing/v1/query.proto

```diff
@@ -122,10 +122,21 @@ message QueryBlsPublicKeyListRequest {
   cosmos.base.query.v1beta1.PageRequest pagination = 2;
 }
 
+// BlsPublicKeyListResponse couples validator address, voting power, and its bls
+// public key
+message BlsPublicKeyListResponse {
+  // validator_address is the address of the validator
+  string validator_address = 1;
+  // bls_pub_key is the BLS public key of the validator
+  bytes bls_pub_key = 2;
```

Comment:
for the output it might make sense to give out as string hex, what do you think?